lib/db/schema/patch-046-047.sql

```diff
@@ -0,0 +1,10 @@
+-- Add ability to get a specific email
```

Comment:
`get a specific email` also could be confusion for future readers, because this is not gonna `get` the primary email